components/04-reference-pages/07-faculty-profile-page/faculty-profile-page.hbs

```diff
@@ -19,20 +19,22 @@
       {{> @breadcrumb }}
     </section>
     <!-- End Slider -->
-
-    <section class="accounting-faculty-sec" id="skipToContent">
+    <section class="position-relative py-5" id="skipToContent">
       <div class="container">
-        <div class="accounting-information-header">
-          <h2>Juan Manuel Sanchez, Ph.D.</h2>
-          <h5>Associate Dean of Graduate Studies and Research
-            Ph.D. Program Director, Professor of Accounting
-          </h5>
-        </div>
-        <div class="accounting-faculty-img-content">
-          <div class="row">
+        <div class="profile-details-component-div white">
+          <div class="col-xxl-12 col-xl-12 col-lg-12 col-md-12 col-sm-12">
```

Comment:
this should only be. **<div class="col-12">**  since all dimensions use the same layout

Reply:
👍